test/core/client_channel/lb_policy/pick_first_test.cc

```diff
@@ -313,8 +305,7 @@ TEST_F(PickFirstTest, ShufflingDisabled) {
         BuildUpdate(kAddresses, MakePickFirstConfig(false)), lb_policy_.get());
     EXPECT_TRUE(status.ok()) << status;
     std::vector<absl::string_view> address_order;
-    GetOrderAddressesArePicked(kAddresses, true, &address_order);
-    EXPECT_THAT(address_order, ::testing::ElementsAreArray(kAddresses));
```

Comment:
This expectation is still needed.  The whole point of this test is to verify that the address list was *not* shuffled.  Without this expectation, the test is not doing that.

Reply:
Reverted